docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec.adoc

```diff
@@ -1 +1,15 @@
-= Managing compliance with Enterprise Contract
\ No newline at end of file
+= Managing compliance with the Enterprise Contract
+
+The Enterprise Contract (EC) is an artifact verifier and customizable policy checker. By default, {ProductName} adds the Enterprise Contract as an integration test to each new application. The Enterprise Contract then keeps your software supply chain secure and ensures container images comply with your organization's policies. It does this by verifying the security and provenance of builds created through {ProductName}.
+
+{ProductName}'s build process uses Tekton Chains to produce a signed in-toto attestation of the build pipeline. The Enterprise Contract then uses this attestation to verify the build's integrity and compliance with a set of policies. These policies include best practices and any organization-specific requirements.
```

Comment:
nit: Do you want to say that it is a provenance attestation?

Reply:
Added to the text, thank you